lib/Frontend/nvqpp/ASTBridge.cpp

```diff
@@ -53,8 +53,14 @@ listReachableFunctions(clang::CallGraphNode *cgn) {
 // Does `ty` refer to a Quake quantum type? This also checks custom recursive
 // types. It does not check builtin recursive types; e.g., `!llvm.ptr<T>`.
```

Comment:
Should update this comment. "custom recursive types" might be better stated as something like CC dialect types? (I wonder just how old this comment is. heh)